cmd/image-builder-db-test/main_test.go

```diff
@@ -26,6 +26,10 @@ const (
 	ORGID2 = "100001"
 	ORGID3 = "100002"
 
+	EMAIL1 = "user1@test.test"
+	EMAIL2 = "user2@test.test"
```

Comment:
EMAIL 2 and 3 aren't actually used :sweat_smile:

Reply:
But three is a lucky number :shrug: 
I think I've added the constants first and then just used the same number as ANR and ORGID, forgot to check if I actually use all three. It's fixed now :sweat_smile: